Data/Scripts/GardenConquest/Core/ConquestSettings.cs

```diff
@@ -28,6 +28,7 @@ public struct SETTINGS {
 			public List<ControlPoint> ControlPoints { get; set; }
 			public int CPPeriod { get; set; }
 			public int CleanupPeriod { get; set; }
+			public bool CommandsRequireClassifier { get; set; }
```

Comment:
Would you please rename this to `SimpleOwnership`? We need it to indicate that we use classifier for ownership checks not just for commands but for `GridEnforcer.reevaluateOwnership()` as well. We want to do the latter to take advantage of the fact that thieves have to grind classifiers now, so we can do the ownership flip immediately there, instead of people saying "Someone stole my ship but it's still counting against my limits!.